src/components/sidebar/sidebar.component.tsx

```diff
@@ -16,6 +16,7 @@ interface SidebarProps {
   onCancel: () => void;
   handleClose: () => void;
   hideFormCollapseToggle: () => void;
+  isSubmissionTriggeredExternally?: boolean;
```

Comment:
I would name this something like `hideControls` or `hideInternalButtons` which defaults to `false`.